src/figuresClasses.ts

```diff
@@ -1,11 +1,78 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: 'red' | 'green' | 'blue';
```

Comment:
better to create some type for Color and reuse that
```suggestion
  color: Color;
```